thetis/inversion_tools.py

```diff
@@ -49,6 +50,7 @@ def __init__(self, sta_manager, output_dir='outputs', no_exports=False, real=Fal
         self.test_gradient = test_gradient
         self.outfiles_m = []
         self.outfiles_dJdm = []
+        self.control_exporters = []
```

Comment:
We could plausibly use `ExportManager` to handle all of these outputs. The user could then decide whether to export these in vtk or hdf5 format.

Reply:
Makes sense.